simplq/src/api/auth.js

```diff
@@ -25,7 +25,7 @@ const getAuthHeaderValue = async (auth) => {
     localStorage.setItem(ANONYMOUS_DEVICE_ID_KEY, `anonymous-${uuidv4()}`);
   }
 
-  return Promise.resolve(`Anonymous ${localStorage.getItem(ANONYMOUS_DEVICE_ID_KEY)}`);
+  return `Anonymous ${localStorage.getItem(ANONYMOUS_DEVICE_ID_KEY)}`;
 };
```

Comment:
The async function already wraps the return value in a promise by default, so felt this wasn't necessary.

Reply:
Good catch, it was a bug